driver/flambda_backend_args.ml

```diff
@@ -567,6 +567,13 @@ let mk_restrict_to_upstream_dwarf f =
 let mk_no_restrict_to_upstream_dwarf f =
   "-gno-upstream-dwarf", Arg.Unit f, " Emit potentially more DWARF information than the upstream compiler"
 
+let mk_dwarf_inlined_frames f =
+  "-gdwarf-inlined-frames", Arg.Unit f, "Emit DWARF inlined frame information"
```

Comment:
(The document string should start with a space.)

Reply:
Fixed